.husky/pre-commit

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 
-npx pretty-quick --staged
+npx lint-staged
```

Comment:
Maybe change to `npx -y` so it doesn't expect user input if the package isn't installed? Many graphical git clients will either consider it failed or just wait for a timeout, if even that.